homeassistant/components/logbook/__init__.py

```diff
@@ -186,6 +187,39 @@ def json_events():
         return await hass.async_add_job(json_events)
 
 
+class LogbookTimeSliceView(HomeAssistantView):
+    """Handle logbook view requests."""
+
+    url = "/api/logbook_timeslice"
+    name = "api:logbook_timeslice"
+
+    def __init__(self, config):
+        """Initialize the logbook view."""
+        self.config = config
+
+    async def get(self, request):
+        """Retrieve logbook entries."""
+        start_timestamp = request.query.get("start_timestamp")
+        if start_timestamp is None:
+            return self.json_message("Invalid start_timestamp", HTTP_BAD_REQUEST)
+        end_timestamp = request.query.get("end_timestamp")
+        if end_timestamp is None:
+            return self.json_message("Invalid end_timestamp", HTTP_BAD_REQUEST)
+        entity_id = request.query.get("entity")
+        hass = request.app["hass"]
+
+        start_time = datetime.fromtimestamp(float(start_timestamp))
+        end_time = datetime.fromtimestamp(float(end_timestamp))
```

Comment:
We should use iso8601 timestrings and parse them with `dt_util.parse_datetime(datetime)` like in the other view.

Any reason this should be it's own view instead of adding support for an end timestamp to the other view ? We could allow sending in either `period` or `end`

I know that the API of the other view is very ugly, it once started beautiful with `/api/logbook/2020-05-26` before we realized that dates start on different times in different timezones and had to make it a timestamp 😬